http/server.lua

```diff
@@ -746,6 +773,11 @@ local function process_client(self, s, peer)
     while true do
         local hdrs = ''
 
+        connections[fiber.id] = fiber.clock()
```

Comment:
I guess if options is not enabled we shouldn't call fiber.clock at all.

Reply:
Fixed.

```lua
--- a/http/server.lua
+++ b/http/server.lua
@@ -773,8 +773,8 @@ local function process_client(self, s, peer)
     while true do
         local hdrs = ''
 
-        connections[fiber.id] = fiber.clock()
         if self.options.idle_timeout > 0 then
+            connections[fiber.id] = fiber.clock()
             close_idle_connections(connections, self.options.idle_timeout)
         end
 
```